src/PlaceholderComponent.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import reactStringReplace from 'react-string-replace';
+
+
+function PlaceholderTextComponent({ title }) {
+  return (
+    <React.Fragment>
+      {reactStringReplace(title.text, title.regex, (match, i) => (
+        <span key={i} className={title.cssClass} />
+      ))
+    }
+    </React.Fragment>
+  );
+}
+
+PlaceholderTextComponent.propTypes = {
+  title: PropTypes.oneOfType([
```

Comment:
this is confusing... your render function uses a single title prop but always assumes its an object with nested text, regex and cssClass attributes... this seems inconsistent with that.
In saying this, as discussed at the top of this class, we originally agreed on 3 props being passed in